tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/InitConfigureProjectPage.java

```diff
@@ -1526,7 +1526,8 @@ else if( bundUrl != null && bundUrl.length() > 0 && !bundleUrlValidation.compute
 
                 if( dataModel.getImportFinished().content() )
                 {
-                    message = "import finished, please go to first page and click \"Restart...\"";
+                    message =
+                        "import finished, if you want to re-import, please go to first page and click \"Restart...\"";
```

Comment:
Change message to 
`"Import has finished.  If you want to reimport, please go to the first page and click "Restart..."`